libraries/AP_GPS/GPS_Backend.h

```diff
@@ -53,8 +53,8 @@ class AP_GPS_Backend
     virtual void handle_msg(const mavlink_message_t *msg) { return ; }
     virtual void handle_gnss_msg(const AP_GPS::GPS_State &msg) { return ; }
 
-    // driver specific lag
-    virtual float get_lag(void) const { return 0.2f; }
+    // driver specific lag, returns true if the driver is confident in the provided lag
+    virtual bool get_lag(float &lag) const { lag = 0.2f; return true; }
```

Comment:
This should assume the normal/more expected lag term. The rule of thumb is that the lag is somewhere around one measurement interval, which makes 0.2 a reasonable value as we are expecting them to be using. This is used for all the drivers so changing it would be a behavior change (and anytime a driver is added which is worse would change every other drivers).